### YamlMime:PythonClass
uid: azure.mgmt.rdbms.mysql.v2020_01_01.models.CloudErrorAutoGenerated
name: CloudErrorAutoGenerated
fullName: azure.mgmt.rdbms.mysql.v2020_01_01.models.CloudErrorAutoGenerated
module: azure.mgmt.rdbms.mysql.v2020_01_01.models
inheritances:
- msrest.serialization.Model
summary: An error response from the Batch service.
constructor:
  syntax: 'CloudErrorAutoGenerated(*, error: typing.Union[_ForwardRef(''ErrorResponse''),
    NoneType] = None, **kwargs)'
  parameters:
  - name: error
    description: Error message.
    types:
    - <xref:azure.mgmt.rdbms.mysql.v2020_01_01.models.ErrorResponse>
